### ISSUE TITLE
week_2 seminar.ipynb

### ISSUE THREAD

**Issue by aulasau** [yandexdataschool/nlp_course#103]
Problem with this [seminar](https://github.com/yandexdataschool/nlp_course/blob/2022/week02_classification/seminar.ipynb).
Right now error is raised in cells with `model.predict(make_batch(...))`, because there is no `predict` method for nn.Module class.
Please, add somewhere function like:
```python
def predict(model, batch):
    return model(batch).detach().numpy()
```
and change `model.predict(make_batch(...).detach().cpu()` expression to the `predict(model, make_batch(...)))`.

**Comment by poedator** [comment]
while this comment may be valid, it does not seem applicable to the current version of the seminar notebook. Feel free to submit a PR if still see a need for similar improvement.
